cart-management/cart.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://glow-css.netlify.app/component.css">
+    <link rel="stylesheet" href="/cart-management/cart.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <title>Kstyle | Cart</title>
+</head>
+
+<body>
+    <!-- NavBar -->
+    <nav id="top">
+        <ul class="nav-list">
+            <li class="nav-item nav-brand"><a href="/index.html" class="nav-link link">KStyle</a></li>
+        </ul>
+        <ul class="nav-list">
+            <li class="nav-item"><input type="text" class="icon" placeholder="search here"></li>
+        </ul>
+        <ul class="nav-list">
+            <li class="nav-item"><a href="/wishlist.html" class="nav-link"><span class="badge badge-span">
+                        <i class="fa fa-heart fa-x"></i>
+                        <span class="badge-circle badge-right"> 2 </span>
+                    </span></a></li>
+            <li class="nav-item"><a href="/cart.html" class="nav-link link"><span class="badge badge-span">
+                        <i class="fa fa-cart-plus fa-x"></i>
+                        <span class="badge-circle badge-right"> 0 </span></span> Cart
```

Comment:
It will be good, if you could write normal text in new line , separated from html codes